deploy/crds/nodemaintenance_crd.yaml

```diff
@@ -64,6 +64,7 @@ spec:
               format: int64
               type: integer
           type: object
+      type: object
```

Comment:
Isn't this part of another PR that was already merged?
You should rebase this one (do not start a new PR) onto the current master.

Reply:
Rebased it now; Didn't do that previously as the github io told me that there are no conflicts with the main branch.